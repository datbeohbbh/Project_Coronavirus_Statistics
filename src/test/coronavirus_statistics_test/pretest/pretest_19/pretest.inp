39 20
q 1 13 121
q 1 2 482
? 4 1
q 1 29 89
? 7
q 1 35 222
? 6
? 7
q 1 3 268
? 4 4
? 7
? 4 1
? 4 4
? 4 1
q 3 2 301
q 1 7 244
? 4 4
? 7
? 7
? 7