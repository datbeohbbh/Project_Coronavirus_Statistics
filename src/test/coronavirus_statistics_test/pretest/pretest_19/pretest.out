Query ? 4 1 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 121
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 4 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 268
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 89
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 268
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 89
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 222
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 301 people

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 301 people

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 301 people

